renku/cli/dataset.py

```diff
@@ -243,6 +243,18 @@ def unlink(client, name, include, exclude, yes):
         click.secho('OK', fg='green')
 
 
+@dataset.command('rm')
+@click.argument('names', nargs=-1)
+@pass_local_client(clean=True, commit=True)
+def remove(client, names):
+    """Delete a dataset."""
+    for name in names:
+        dataset = client.load_dataset(name=name)
+        if dataset:
+            client.remove_dataset(dataset)
+            click.secho('OK', fg='green')
```

Comment:
```suggestion
    click.secho('OK', fg='green')
```

Reply:
I think we should not print anything if no dataset is found. On unlink, we are handling this by not outputting anything so we should handle it in the same way here or we do a proper message for such cases on both commands.

Reply:
Maybe we can start with removing a single dataset first and then refactor the name resolution to something like:

```
datasets = {
    datasets.__reference__: dataset
    for dataset in (client.load_dataset(name) for name in names)
    if dataset
}
with progressbar(datasets.values()) as bar:
    for dataset in bar:
        client.remove_dataset(dataset)
click.secho('OK', fg='green')

Reply:
@jsam 🙄 ⬆️ ❓

Reply:
Ah, yes! This is another issue now. How do you see this solution of resolving by name? Since the paths are uuid based now we cannot simply query the filesystem and get that answer. Do we have some additional mapping somewhere with which this can be done?

Reply:
Ah, I see we could do LinkReference. Ok!